daemon/compartment.c

```diff
@@ -782,6 +789,18 @@ compartment_start_child(void *data)
 		goto error;
 	}
 
+	DEBUG("Executing start_pre_exec_child_early hooks");
```

Comment:
introduce new start_pre_exec_child_early() hook in separate commit

Reply:
done